src/router/index.js

```diff
@@ -140,7 +146,13 @@ const routes = [
     path: '/signin',
     name: 'SignIn',
     component: () => import('../pages/SignIn.vue'),
-    meta: { pageTitle: 'Sign In' },
+    meta: {
+      pageTitle: {
+        'en-US': 'Sign In',
+        en: 'Sign In',
+        es: 'Iniciar Sesión',
+      },
+    },
```

Comment:
See comment above about moving all translation content to the `translations` directory.